include/itkMorphologicalContourInterpolator.h

```diff
@@ -239,7 +239,7 @@ class MorphologicalContourInterpolator:
   ImagesEqual( typename BoolSliceType::Pointer& a, typename BoolSliceType::Pointer& b );
 
   /** Does the real work. */
-  virtual void
+  void
   GenerateData() ITK_OVERRIDE;
```

Comment:
why remove the keyword `virtual`?

Reply:
There is already a link in the commit message :smile:
https://stackoverflow.com/questions/39932391/virtual-override-or-both-c